Contributing.md

```diff
@@ -34,12 +34,25 @@ Additionally, changes to message definitions are expected to include bag migrati
 ### Styleguide
 
 * Message field names are lowercase, with words separated by underscore. (from the [ROS style guide](http://wiki.ros.org/ROS/Patterns/Conventions))
-* Please configure your editor to strip trailing whitespace
+* We use pre-commit to check for formmating and styling:
```

Comment:
Yay! Thanks for updating the README. This made it really easy for me to test out these improvements =)